README.md

```diff
@@ -86,6 +86,23 @@ Afterwards, the web application will be presented on http://localhost:8000
 
 ---
 
+## How To Remove The Local Project
```

Comment:
I think you can edit this to `## How To Remove The Project`.

Reply:
Done